src/pseudopeople/configuration/generator.py

```diff
@@ -76,19 +76,23 @@ def get_configuration(user_configuration: Union[Path, str, Dict] = None) -> Conf
     :return: a ConfigTree object of the noising configuration
     """
 
-    if type(user_configuration) == str and user_configuration.lower() == Keys.NO_NOISE:
-        config_type = Keys.NO_NOISE
+    if isinstance(user_configuration, str) and user_configuration.lower() == NO_NOISE:
+        config_type = NO_NOISE
         user_configuration = None
+    elif isinstance(user_configuration, (Path, str)):
+        with open(user_configuration, "r") as f:
+            user_configuration = yaml.full_load(f)
+        config_type = None
     else:
-        config_type = Keys.DEFAULT
+        config_type = None
     noising_configuration = _generate_configuration(config_type)
     if user_configuration:
         add_user_configuration(noising_configuration, user_configuration)
 
     return noising_configuration
 
 
-def _generate_configuration(config_type: str) -> ConfigTree:
+def _generate_configuration(config_type: Optional[str]) -> ConfigTree:
```

Comment:
Instead of a string `config_type`, I'd suggest a boolean argument `is_no_noise`. That would be cleaner and avoid the need to compare strings.